test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,37 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+//I believe we are writing to servoPin, which I set to be PA_1
+CAN can1(PA_0, servoPin);
+float pulseWidthToSend;
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2) {
+    m_servoPwmOut = servoPin;
+    m_servoRangeInDegrees = servoRangeInDegrees;
+    m_minPulsewidthInMs = minPulsewidthInMs;
+    m_maxPulsewidthInMs = maxPulsewidthInMs;
+}
+
+// Set servo position (ex. 45 deg)
+void setPositionInDegrees( const float degrees) {
+    m_servoPwmOut.period_ms(20);
```

Comment:
This can go in the constructor

Reply:
Fixed in 0fabe3f72fab676fa8c9a4c6ba2574a79b1cc32f

Reply:
Sorry, I removed it by accident when doing another change. It's fixed fully in 4484876e338b503670098838e50fec327e357c6c